internal/capability/operator_install_test.go

```diff
@@ -0,0 +1,99 @@
+package capability
+
+import (
+	"context"
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/opdev/opcap/internal/operator"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Operator install tests", func() {
+	var csv operatorv1alpha1.ClusterServiceVersion
+	var fakeClient operator.Client
+	var operatorGroupData operator.OperatorGroupData
+	var subscription operator.SubscriptionData
+	var customResources []map[string]interface{}
+	var auditFn auditFn
+	var auditCleanupFn auditCleanupFn
+	BeforeEach(func() {
+		// Set up and clean up temporary directory for CSV created in operator_install
+		tmpDir, err := os.MkdirTemp("", "operator-install-*")
```

Comment:
why do you need a temp dir here ?

Reply:
It's because currently the operator install audit writes it's json file to the current working directory. It is undesirable for a unit test to leave behind artifacts like that. So, temp dir while the test runs, then throw it away.

Reply:
ah i see.. was thinking we could use afero

Reply:
Eventually, that's the idea. I've got that in another PR.